content/desktop/hardened-desktop/settings-management/configure.md

```diff
@@ -138,22 +138,40 @@ The following `admin-settings.json` code and table provides an example of the re
   "blockDockerLoad": {
     "locked": false,
     "value": true
+  },
+	"filesharingAllowedDirectories": [
+	  {
+	    "path": "$HOME",
+	    "sharedByDefault": true
+	  },
+	  {
+	    "path":"$TMP",
+	    "sharedByDefault": false
+	  }
+	], 
```

Comment:
nit: looks like there's a mix of tabs and spaces in the indentation here, which could lead to strange behavior when rendered. I suggest we use spaces
```suggestion
  "filesharingAllowedDirectories": [
    {
      "path": "$HOME",
      "sharedByDefault": true
    },
    {
      "path":"$TMP",
      "sharedByDefault": false
    }
  ], 
```